test-clients/src/main/java/com/hedera/services/legacy/netty/MultiAccountTransferSeqThread.java

```diff
@@ -337,25 +330,22 @@ public void doOneIteration(int _acItr)
 
         try{
           transferRes = stub.cryptoTransfer(transfer1);
-     //     transferRes = stub.withDeadlineAfter(deadLineMs, TimeUnit.MILLISECONDS).cryptoTransfer(transfer1);
         }catch(Exception dex) {
           log.error("Skipping this due to " + dex.getMessage());
         }
 
         if ((transferRes != null) && (ResponseCodeEnum.OK == transferRes.getNodeTransactionPrecheckCode())) {
         TransactionBody transferBody = TransactionBody.parseFrom(transfer1.getBodyBytes());
           transList.add(transferBody.getTransactionID());
-
-          //transferMap.put(transferBody.getTransactionID(), transfer1);
         } else
           {
           log.error("Got Bad Precheck Stuff ** Adding to Retry Queue ** " + transferRes
               .getNodeTransactionPrecheckCode());
           if(transferRes.getNodeTransactionPrecheckCode() == ResponseCodeEnum.TRANSACTION_OVERSIZE){
             badTransList.add(transfer1);
           }
-          else totalBadReceipts++;
-         // totalBadReceipts++;
+          else {
+          }
```

Comment:
Should the change be like this:

```
else {
    totalBadReceipts++;
}
```

Reply:
This counter is no more. Removing this empty else block